exasol_script_languages_container_ci_setup/health_check.py

```diff
@@ -0,0 +1,79 @@
+"""
+The health check module provides functionality to check the health of the `slc-ci-setup`
+package and also provide help to find potential fixes.
+"""
+import shlex
+import subprocess
+import sys
+from typing import Iterator, Optional
+
+from exasol_error_reporting_python.error_message_builder import ErrorMessageBuilder
+from exasol_error_reporting_python.exa_error import ExaError
+
+SUPPORTED_PLATFORMS = ["linux", "darwin"]
+
+
+def check_shell_cmd(cmd: str) -> bool:
+    """
+    Runs a shell command and returns True if exit code was 0, False otherwise 
+    :param cmd: shell command to execute
+    :return: returns True if exit code was 0, False otherwise
+    """
+    result = subprocess.run(shlex.split(cmd), stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    return result.returncode == 0
+
+
+def is_supported_platform(**kwargs) -> Optional[ErrorMessageBuilder]:
+    """
+    Checks weather or not the current platform is supported.
+    """
+    if sys.platform not in SUPPORTED_PLATFORMS:
+        return ExaError.message_builder('E-SLCCS-02') \
+            .message("The platform you are running on is not supported.") \
+            .mitigation("Make sure you are using one of the following platforms: {SUPPORTED_PLATFORMS}.")
+
+
+def aws_cli_available(**kwargs) -> Optional[ErrorMessageBuilder]:
```

Comment:
do we still need the AWS CLI?

Reply:
Hm, we need it if one needs to analyze a possible problem during stack deployment. Under the hood we still need the credentials, config, .....so maybe it's easier to keep it.

Reply:
Can we add it to poetry?

Reply:
As you already pointed out (https://github.com/aws/aws-cli/issues/4947), unfortunately not.